core-api/src/main/java/org/silverpeas/core/date/period/Period.java

```diff
@@ -44,7 +44,9 @@
  * <p>
  * User: Yohann Chastagnier
  * Date: 19/04/13
+ * @deprecated Use {@link org.silverpeas.core.date.Period}
  */
+@Deprecated
 public class Period implements Comparable, Serializable, Cloneable {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)